src/common/DataUtils.hpp

```diff
@@ -160,10 +160,24 @@ long double calcChecksum(Complex_ptr d, Size_type len,
 
 
 /*!
- * \brief Get an host accessible data space for this dataSpace.
+ * \brief Get an host based data space for the given dataSpace.
```

Comment:
Similar comment about whether this is less clear.

Reply:
This is a good thing to discuss. I added this distinction to be able to differentiate between memory spaces with good performance when running on the host vs those that are accessible but far away, like across a pcie on another device. Is the naming or the idea unclear, or is the distinction more specific than we want to have?

Reply:
I think the idea is clear. I think the naming and comments aren't adequately clear to know what's going on without poking around in the code to see what it does.

Reply:
Ok, let's iterator on that.

Reply:
Works for me!